packages/sit-onyx/src/components/OnyxIcon/OnyxIcon.vue

```diff
@@ -28,6 +28,7 @@ const props = withDefaults(defineProps<OnyxIconProps>(), {
 
   display: inline-block;
   width: var(--icon-size);
+  min-width: var(--icon-size);
```

Comment:
for clarification: I suppose there is a reason to add this, but is it not needed for min-height as well?

Reply:
Yes its needed so the icon does not shrink/get squeezed when used in a flex container.
Its not needed for the height because the icon is squared so setting either width or height is enough

Reply:
ok I didn't see aspect-ratio so I was not sure if it is enough that the file itself is squared.